src/components/m-datepicker/index.js

```diff
@@ -56,8 +54,8 @@ class AXADatepicker extends BaseComponentGlobal {
    */
   connectedCallback() {
     super.connectedCallback();
-
     this.className = `${this.initialClassName} m-datepicker`;
+    this.datepicker = new Datepicker(this);
```

Comment:
awesome, though be aware of the `connectedCallback` API and that it can potentially be called more than once...

Reply:
I removed all datepicker changes. they were not meant to be in this PR.